tests/modules/collaborations/resources/test_collaboration_patch.py

```diff
@@ -0,0 +1,59 @@
+# -*- coding: utf-8 -*-
+# pylint: disable=missing-docstring
+import tests.modules.collaborations.resources.utils as collab_utils
+from tests import utils
+
+
+def test_patch_collaboration(
+    flask_app_client, researcher_1, readonly_user, researcher_2, db
+):
+    from app.modules.collaborations.models import Collaboration
+
+    collab = None
+    try:
+        data = {'user_guid': str(researcher_2.guid)}
+        collab_utils.create_collaboration(flask_app_client, researcher_1, data)
+        collabs = Collaboration.query.all()
+        collab = collabs[0]
+        collab_guid = collab.guid
+        assert not collab.user_has_read_access(researcher_1.guid)
+        assert collab.user_has_read_access(researcher_2.guid)
+
+        # should work
+        patch_data = [utils.patch_replace_op('title', 'Some sort of title')]
+        collab_utils.patch_collaboration(
+            flask_app_client,
+            collab_guid,
+            researcher_2,
+            patch_data,
+        )
+        assert collab.title == 'Some sort of title'
```

Comment:
We probably don't even need title, that's the old boilerplate cruft that got added to all models prior to being cut.

Reply:
If it's cruft, it's gone, new push has removed it